ocs_ci/ocs/resources/pod.py

```diff
@@ -1661,6 +1661,10 @@ def check_pods_in_running_state(
             ("rook-ceph-osd-prepare" not in p.name)
             and ("rook-ceph-drain-canary" not in p.name)
             and ("debug" not in p.name)
+            and not (
+                config.ENV_DATA["platform"] in constants.MANAGED_SERVICE_PLATFORMS
+                and p.name[:-5] in p.labels.get("job-name", "")
```

Comment:
@jilju are you sure that `p.name[:-5]` not too hardcoded and error prone?

Reply:
@ebenahar Yes, for that particular kind of pod.

Instead, can we "Completed" pods ?